web_src/css/repo.css

```
@@ -2417,6 +2417,10 @@
   gap: 0 !important;
 }
 
+.archived-label {
+  filter: grayscale(1);
```

Sounds a bit harsh, what about
```suggestion
  filter: grayscale(0.7);
```
instead?

---

I personally revere 100% as it's easier to spot by my :eyes:

---

Let me put it that way:
In your screenshot, I didn't even notice at first that there is an archived label.
The gray looked like it is the actual label color.
Plus, having this tiny bit of color helps me at least with visual orientation.
Have edited my message now to 80%.